core/embed/rust/src/ui/component/text/layout.rs

```diff
@@ -107,7 +113,13 @@ impl TextLayout {
         cursor: &mut Point,
         sink: &mut dyn LayoutSink,
     ) -> LayoutFit {
-        let init_cursor: Point = *cursor;
+        let init_cursor = *cursor;
+        let height = move |end_cursor: &Point| {
+            self.top_space
+                + self.text_font.text_height()
+                + (end_cursor.y - init_cursor.y)
+                + self.bottom_space
+        };
```

Comment:
Maybe it would be better to put this into a method? Something like `fn layout_height(&self, init_cursor: Point, end_cursor: Poin) -> i32`?

Reply:
Fixed in f24dca9. The text height might be different at the end compared to the beginning but we don't really support mixing font sizes anyway.